src/main/scala/org/graphframes/GraphFrame.scala

```diff
@@ -912,4 +912,12 @@ object GraphFrame extends Serializable with Logging {
     _broadcastThreshold = value
     this
   }
+
+  /**
+   * Return the given DataFrame with columns in the order specified by `names`.
+   * @param names This method assumes that names is exactly the set of columns in the DataFrame.
```

Comment:
The semantics of the method seem slightly different, it seems to assume that columns is a subset of those in Dataframe

Reply:
The doc doesn't explain the behavior when `names` is empty and it is not clearly stated by the method name.

Reply:
Actually, I forgot to remove this; it used to be used in several places.

Reply:
Actually I might eliminate this.  It used to be used in several places but now only 1 (in the other PR).  There's not much value in keeping it as its own method now.